Frends.Instagram.Request/Frends.Instagram.Request/Frends.Instagram.Request.cs

```diff
@@ -0,0 +1,154 @@
+﻿#pragma warning disable SA1200 //Using directives should be placed correctly.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Frends.Instagram.Request.Definitions;
+
+namespace Frends.Instagram.Request;
+
+/// <summary>
+/// Facebook class.
+/// </summary>
+public static class Instagram
+{
+    /// <summary>
+    /// This is task for reading data from Instagram API.
```

Comment:
Please use implicitly Frends Task.
Also this task is not only for reading but send requests.